inductiva/molecules/scenarios/protein_solvation/README.md

```diff
@@ -0,0 +1,44 @@
+# Protein Solvation Scenario
+
+This scenario models the dynamics of a protein solvated in water inside a cubic box. The system is modelled with the motion equations for each individual atoms according to the prescription of a molecular force field approximation.
+
+To initilaize the scenario, the user can define the following parameters:
+-- Protein .pdb file: PDB file containing the structure of the protein to simulate;
+-- Temperature of the system in K.
+
+Now, the user is prepared to simulate protein dynamics. Please note that the scenario currently supports only PDB files as input. We have conducted extensive testing with structures generated by the AlphaFold2 model and structures from the RCSB database. Before simulation, files undergo pre-processing to strip away all non-protein components of the system, such as ligands, non-standard residues, or water molecules, leaving only the protein.
+
+The user can further define the following simulation parameters:
+-- simulator: The simulator to be used for the simulation. Currently, only `GROMACS` is supported.
```

Comment:
Remove this for now, since no other options are supported. Just mention it on top!